charts/aac-manage-case-assignment/values.yaml

```diff
@@ -3,6 +3,9 @@ java:
   image: 'hmctspublic.azurecr.io/aac/manage-case-assignment:latest'
   ingressHost: aac-manage-case-assignment-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   aadIdentityName: aac
+  autoscaling:
+    enabled: true 
+    maxReplicas: 4
```

Comment:
pls let's also include minReplicas.
I suggest 2 for it.

Reply:
default is 2 already